docs/getting-started.md

```diff
@@ -59,20 +59,28 @@ const {ChromaClient} = require('chromadb');
 const client = new ChromaClient();
 ```
 
-To connect to Chroma's backend - you either need to connect to a hosted version of Chroma, or run it on your local computer. If you can run `docker-compose up -d --build` you can run Chroma. 
+</TabItem>
+
+</Tabs>
+
+To connect to Chroma's backend - you either need to connect to a hosted version of Chroma, or run it on your local computer.
```

Comment:
This should be inside the tab for `javascript` since this is not true for `python`

Reply:
whoops - meant to mark this as `Request Changes` cc @buddhikajay

Reply:
@buddhikajay any update here? would love to get this merged in! let me know if you want me to take over the branch